LucidGloves/lucidgloves/resources/settings/default.vrsettings

```diff
@@ -1,12 +1,19 @@
 {
-	"driver_lucidgloves" : {
-		"enable" : true,
+	"driver_lucidgloves": {
+		"enable": true,
 		"protocol": 0,
 		"serial_left_port": "\\\\.\\COM4",
 		"serial_right_port": "\\\\.\\COM5",
 		"x_offset": -0.07,
 		"y_offset": -0.08,
 		"z_offset": 0.07,
-		"pose_offset": -0.06
+		"pose_offset": -0.06,
+		"x_offsetDegrees": 90,
+		"y_offsetDegrees": 0,
+		"z_offsetDegrees": 0,
+		"leftFlippedX": true,
+		"leftFlippedY": false,
+		"leftFlippedZ": true
```

Comment:
Can these have the casing format as the other properties here?
Happy to switch to camelCase if you want.

Reply:
Oops, lol, will switch to underscores